thefuck/rules/git_hook_bypass.py

```diff
@@ -0,0 +1,28 @@
+from thefuck.utils import replace_argument
+from thefuck.specific.git import git_support
+
+hooked_commands = ("am", "commit", "push")
+
+
+@git_support
+def match(command):
+    return any(
+        hooked_command in command.script_parts for hooked_command in hooked_commands
+    )
+
+
+@git_support
+def get_new_command(command):
+    hooked_command = next(
+        hooked_command
+        for hooked_command in hooked_commands
+        if hooked_command in command.script_parts
+    )
+    return replace_argument(
+        command.script, hooked_command, hooked_command + " --no-verify"
+    )
+
+
+priority = 900
+enabled_by_default = True
```

Comment:
This can be omitted as `True` is the default value for it:

https://github.com/nvbn/thefuck/blob/6975d30818792f1b37de702fc93c66023c4c50d5/thefuck/types.py#L144

```suggestion
```

Reply:
Done